src/codemagic/tools/firebase_app_distribution/arguments.py

```diff
@@ -0,0 +1,59 @@
+from codemagic import cli
+from codemagic.cli.colors import Colors
+from codemagic.firebase.resource_managers.resource_manager import ResourceManager
+
+from .argument_types import CredentialsArgument
+
+
+class FirebaseArgument(cli.Argument):
+    PROJECT_ID = cli.ArgumentProperties(
+        key='project_id',
+        flags=('--project-id', '-p'),
+        description=f'Project ID in Firebase. For example {Colors.WHITE("228333310124")}',
```

Comment:
To format the example as code in generated markdown docs, then it would have to be wrapped into backticks.
```suggestion
        description=f'Project ID in Firebase. For example `{Colors.WHITE("228333310124")}`',
```